runtime/lua/vim/filetype.lua

```diff
@@ -770,6 +770,14 @@ local extension = {
   mm = function() vim.fn["dist#ft#FTmm"]() end,
   mms = function() vim.fn["dist#ft#FTmms"]() end,
   p = function() vim.fn["dist#ft#FTprogress_pascal"]() end,
+  patch = function(path, bufnr)
+    local firstline = getline(bufnr, 1)
+    if string.find(firstline, "^From " .. string.rep("%x", 40) .. "+ Mon Sep 17 00:00:00 2001$") then
```

Comment:
I've no idea what this is about, but the date somehow looks suspicious. Are you sure that's what should be here?

Reply:
[Yes](https://github.com/neovim/neovim/blob/master/runtime/filetype.vim#L496)

Reply:
Well then :)